dist/tools/vera++/check.sh

```diff
@@ -29,5 +30,37 @@ fi
 
 VERA_CMD="vera++ --root $CURDIR --exclusions $CURDIR/exclude $_QUIET"
 
+# sets
+# - LOG to tee output into for later parsing
+# - LOGFILE to parse GitHub annotations into
+github_annotate_setup
+
+RESULT=0    # remove if script should error
+
 echo "$FILES" | $VERA_CMD --profile riot \
-    --parameters $CURDIR/profiles/riot_params.txt -w
+    --parameters $CURDIR/profiles/riot_params.txt -w 2>&1 | ${LOG}
+# Uncomment when the script should error
+# RESULT=${PIPESTATUS[0]}
+
+if github_annotate_is_on; then
+    _annotate() {
+        FILENAME=$(echo "${line}" | cut -d: -f1)
+        LINENUM=$(echo "${line}" | cut -d: -f2)
+        DETAILS=$(echo "${line}" | cut -d: -f3- |
+                  sed -e 's/^[ \t]*//' -e 's/[ \t]*$//')
+
+        # parse if warning (errors don't have a label)
```

Comment:
Note that errors do have an "error" label. Since we pass `-w` to the vera++ invocation, we see all errors as warnings

Reply:
See how the output changes depending of the flag (`-w` or `-e`)
```
riotbuild@65e50cd10c65:~/riotbase$ vera++ --root $(pwd)/dist/tools/vera++ --profile riot_force -e sys/net/gnrc/link_layer/gomach/gomach.c
sys/net/gnrc/link_layer/gomach/gomach.c:2205: error: source file is too long
riotbuild@65e50cd10c65:~/riotbase$ vera++ --root $(pwd)/dist/tools/vera++ --profile riot_force -w sys/net/gnrc/link_layer/gomach/gomach.c
sys/net/gnrc/link_layer/gomach/gomach.c:2205: warning: source file is too long
```

Reply:
> Note that errors do have an "error" label. Since we pass `-w` to the vera++ invocation, we see all errors as warnings

But see my [run with the `-w` removed](https://github.com/RIOT-OS/RIOT/pull/15753/checks?check_run_id=1689822766#step:5:29). Here the error does not have an error label. Anyway, I will test with `-e` and adopt if necessary!

Reply:
Ok. I added removing the `error:` label below. [Here is the run without the `-e` flag](https://github.com/RIOT-OS/RIOT/runs/1690123507) and [here it is with the `-e` flag](https://github.com/RIOT-OS/RIOT/pull/15753/checks?check_run_id=1690126292).

Reply:
See https://github.com/RIOT-OS/RIOT/runs/1690123507#step:5:28 vs https://github.com/RIOT-OS/RIOT/pull/15753/checks?check_run_id=1690126292#step:5:28

Reply:
ah yes, removing the `-e` flag doesn't show the error labels. That's a weird behavior, but it is what it is :man_shrugging: